cli/cmd/root.go

```diff
@@ -49,7 +49,10 @@ upgrade, and delete functionality.`,
 
 	// hidden flags
 	root.PersistentFlags().StringVar(&cfg.bcloudServer, "bcloud-server", "https://buoyant.cloud", "Buoyant Cloud server to retrieve manifests from (for testing)")
+	root.PersistentFlags().StringVar(&cfg.bcloudAPI, "bcloud-api", "https://api.buoyant.cloud", "Buoyant Cloud API to retrieve manifests from (for testing)")
```

Comment:
the default needs to be `api.buoyant.cloud:443`. `bcloud-server` is used by `check` and `dashboard` to connect to web URLs, `bcloud-api` is used by install as a `Host` field in a `url.URL`.